README.md

```diff
@@ -15,8 +15,64 @@ npm install glsl-sandbox --save
 ```
 
 ## Usage
+```js
+import { WebGLRenderer, PerspectiveCamera, Vector3 } from 'three';
+import { GlslSandbox } from 'glsl-sandbox';
 
-TODO
+const renderer = new WebGLRenderer();
+const camera = new PerspectiveCamera(45, window.innerWidth / window.innerHeight, 0.01, 100);
+
+const fragmentShader = `
+#define PLATFORM_WEBGL
+
+uniform sampler2D   u_scene;
+
+uniform vec2        u_resolution;
+
+void main() {
+    vec4 color = vec4(vec3(0.0), 1.0);
+    vec2 pixel = 1.0 / u_resolution;
+    vec2 st = gl_FragCoord.xy * pixel;
+
+#if defined(BACKGROUND)
+    // This will render a red background behind the 3D
+    color.r = 1.0;
+
+#elif defined(POSTPROCESSING)
+    // Postprocessing pass,
+    // displays the scene directly to the screen
+    color = texture2D(u_scene, st);
+
```

Comment:
it's missing an `#else` case here. Which will be use for the geometry material (3D scene).
And yes, 2D scenes don't use `BACKGROUND`